components/kyma-environment-broker/internal/process/provisioning/skip_step.go

```diff
@@ -0,0 +1,43 @@
+package provisioning
+
+import (
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/broker"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+)
+
+type SkipStep struct {
+	step             Step
+	operationManager *process.ProvisionOperationManager
+}
+
+func NewSkipStep(os storage.Operations, step Step) *SkipStep {
```

Comment:
if the step is skipping for trial plan, please change the name of the step that we know in the main.go what it does.